dcc-portal-ui/app/index.template.html

```diff
@@ -189,9 +190,9 @@ <h1 style="font-size: 4rem">
 <script src="bower_components/d3/d3.min.js"></script>
 <script src="bower_components/Blob/Blob.js"></script>
 <script src="bower_components/FileSaver/FileSaver.min.js"></script>
-<script src="bower_components/showdown/src/showdown.js"></script>
-<script src="bower_components/showdown/src/extensions/table.js"></script>
-<script src="bower_components/x2js/xml2json.js"></script>
+<script src="bower_components/showdown/<%= grunt.config('env') === 'production' ? 'compressed/Showdown.min.js' : 'src/showdown.js' %>"></script>
+<script src="bower_components/showdown/<%= grunt.config('env') === 'production' ? 'compressed/extensions/table.min.js' : 'src/extensions/table.js' %>"></script>
```

Comment:
Well, I thought one of the grunt tasks we use rewrites the html file and replaces the various scripts with a concatenated version. I was wondering if their was enough metadata in the bower dependencies to be able to do all this automagically.